RELEASE-NOTES.txt

```diff
@@ -12,6 +12,7 @@
 * [*] Reader: Add ability to block a followed site. [#20053]
 * [*] Reader: Add ability to report a post's author. [#20064]
 * [*] [internal] Refactored the topic related features in the Reader tab (i.e. following, unfollowing, and search). [#20150]
+* [*] Fix inaccessible block settings within the unsupported block editor [https://github.com/WordPress/gutenberg/pull/48435]
```

Comment:
FYI: It's great to have this change in the raw release notes for future reference, but since it landed late in the release cycle it will not be part of the editorialized release notes.